### ISSUE TITLE
Rule INFO_XML - False Positive - Domain names that use 'Joomla' or a derivative of Joomla

### ISSUE THREAD

**Issue by anibalsanchez** [joomla-extensions/jedchecker#113]
Hi,

Tike the rule that checks for Joomla is overzealous and too strict. In this case, joomcore is an allowed domain. What is not allowed is the literal use of joomla in domains.

```
Domain names that use 'Joomla' or a derivative of Joomla ('http://www.joomcore.com/') need to be licensed by OSM. Please, check your domain name is listed on the Joomla! Trademark Approval Registry page.
```
@dryabov

**Comment by Llewellynvdm** [comment]
Please make the needed PR to relax this.

**Comment by anibalsanchez** [comment]
What do you think @dryabov ? Could you correct the rule as part of the rest of the changes?

**Comment by dryabov** [comment]
The code is based on [TM1 - Domain Violation](https://extensions.joomla.org/support/knowledgebase/error-codes/submission-error-codes/) rule:

> All domains in your listing that use a version of the trademark in the URL, must be registered and approved by Open Source Matters prior to listing approval. Also, the disclaimer required must be clearly visible on your site. This includes domains that have **Joo, Joom, Joomla or something similar** (for example: thejoomlasiteibuiltyesterday.com, thesiteofjoom.com, joodontknowjoomla.com, etc.).

**Comment by anibalsanchez** [comment]
Let me double-check with the TM team.

**Comment by Llewellynvdm** [comment]
This says when you include the **joo** you must have a disclamer on your website. When you use Joomla you need approval and be listed here:  https://tm.joomla.org/approved-domains.html

**Comment by Llewellynvdm** [comment]
So it means you can have **Joo** or **Joom** but with a disclaimer on your website that you are not an official Joomla website. But if it has **Joomla** then your need approval and those approved are listed here https://tm.joomla.org/approved-domains.html

**Comment by anibalsanchez** [comment]
@dryabov At this point is not clear if the rule is too broad. We are looking for clarification and discussing internally how to improve the definition.

To be on the safe side on JED Checker, please, change the validation to restrict domains that include only "Joomla".

**Comment by dryabov** [comment]
I've just pushed a trivial PR that replaces "joom" by "joomla" in `validateDomain`.

**Comment by dryabov** [comment]
@Llewellynvdm You are very fast :-)

**Comment by anibalsanchez** [comment]
Yes, it has to be authorized as well. All uses of the trademark must be authorized.
